src/Parser/BatchTableHierarchyExtensionParser.js

```diff
@@ -0,0 +1,109 @@
+import { $3dTilesAbstractExtension } from '../Provider/3dTilesProvider';
+
+/** Class for storing and accessing information relative to the
+ *  3DTILES_batch_table_hierarchy extension of 3D Tiles */
```

Comment:
Could you add a link to [the specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/extensions/3DTILES_batch_table_hierarchy) ?

Reply:
Ok, done